File: engine/model/src/main/scala/net/server/channel/packet/alliance/AcceptedInvitePacket.scala
package net.server.channel.packet.alliance

import net.server.MaplePacket

class AcceptedInvitePacket( private var _allianceId: Int,  private var _recruitingGuild: String) extends AllianceOperationPacket {
     def allianceId: Int = _allianceId
     def recruitingGuild: String = _recruitingGuild
}

File: engine/engine-model/src/main/java/net/server/channel/packet/alliance/AcceptedInvitePacket.java
package net.server.channel.packet.alliance;

public class AcceptedInvitePacket extends AllianceOperationPacket {
   private final int allianceId;

   private final String recruitingGuild;

   public AcceptedInvitePacket(int allianceId, String recruitingGuild) {
      this.allianceId = allianceId;
      this.recruitingGuild = recruitingGuild;
   }

   public int allianceId() {
      return allianceId;
   }

   public String recruitingGuild() {
      return recruitingGuild;
   }
}
